controllers/restore/controller.go

```diff
@@ -67,6 +69,15 @@ func (r *RepoManagerRestoreReconciler) Reconcile(ctx context.Context, req ctrl.R
 		return ctrl.Result{}, err
 	}
 
+	// if lock configmap is found it means that the restore already ran, so the controller should stop execution.
+	// To rerun a restore the user will have to manually delete the lock configmap first.
+	lockCM := &corev1.ConfigMap{}
```

Comment:
why not use a status?

Reply:
Hum ... I think that the status could be lost in case of the `controller-manager` pod reprovision.
Any idea on how to handle this situation?

Reply:
> "I think we should be more explicit on why this configmap exists"

Even more than
https://github.com/pulp/pulp-operator/blob/e092bcd3db455bf0f2dd41dfc1a47e308a4e5938/controllers/restore/utils.go#L206-L211 ?

Reply:
I meant more for the user

Reply:
Hum... what do you think would be the best way to do it?
Add an explanation in issue #550 or explain it in our documentation or add it to the log?

Reply:
it is more about my comment below, I think we can share a link to the FAQ part